airbyte-webapp/src/hooks/services/EditControls/BulkEditService.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from "react";
+
+const Context = React.createContext<EditControlsServiceContext | null>(null);
+
+export interface EditControlsServiceContext {
```

Comment:
These should probably use the world Connection (for example, ConnectionEditControlsService), and the filename should also have this name over BulkEditService

Reply:
actually, I found that both `EditControls` and `BulkEditPanel` both inside `Formik`, and also `Formik` have something called `status` object, where you can basically pass anything you want that helps you work with form
https://formik.org/docs/api/formik#status-any
```
A top-level status object that you can use to represent a form state that can't otherwise be expressed/stored with other methods. This is useful for capturing and passing through API responses to your inner component.
```
so I decided we can use it